### PR TITLE
LPS-177512 | Investigate failure in ClientExtension#ErrorAppearsWhenHTMLBlank

### PR DESCRIPTION
**Pull request by andreluis2022**
Background:
selecting on the, new custom element (the HTML Element Name field is required).

Plan:
- Given Add Custom Element Client Extensions admin page
- When required fields Name, HTML Element Name, and JS URL are blank and When attempt to save remote app
- Then error message 'HTML element name is empty
- And Then Remote App is not added

JIRA Ticket:
https://issues.liferay.com/browse/LPS-177512

### CODE REVIEW

**Comment by mateusralv** [comment]
ci:test:frontend

**Comment by mateusralv** [comment]
Hi @andreluis2022 your solution fix the [LocalFile.ClientExtension#ErrorAppearsWhenHTMLBlank](https://testray.liferay.com/home/-/testray/case_results/2223803799?redirect=https%3A%2F%2Ftestray.liferay.com%2Fhome%2F-%2Ftestray%2Fcase_results%2Findex%3FtestrayBuildId%3D2223801156%26testrayRunId%3D2223801860%26cur%3D2%26delta%3D200%26orderByCol%3Dstatus_sortable%26orderByType%3Dasc%26resetCur%3Dfalse), but broke others, could you take a look?  🥲 

[LocalFile.ClientExtension#HTMLElementConatinsInvlaidCharacters](https://testray.liferay.com/home/-/testray/case_results/2223803808?redirect=https%3A%2F%2Ftestray.liferay.com%2Fhome%2F-%2Ftestray%2Fcase_results%2Findex%3FtestrayBuildId%3D2223801156%26testrayRunId%3D2223801860%26cur%3D1%26delta%3D200%26orderByCol%3Dstatus_sortable%26orderByType%3Dasc%26resetCur%3Dfalse)
[LocalFile.ClientExtension#HTMLElementMustHaveOneHyphen](https://testray.liferay.com/home/-/testray/case_results/2223802766?redirect=https%3A%2F%2Ftestray.liferay.com%2Fhome%2F-%2Ftestray%2Fcase_results%2Findex%3FtestrayBuildId%3D2223801156%26testrayRunId%3D2223801860%26cur%3D1%26delta%3D200%26orderByCol%3Dstatus_sortable%26orderByType%3Dasc%26resetCur%3Dfalse)
[.ClientExtension#HTMLElementMustStartWithLowercase](https://testray.liferay.com/home/-/testray/case_results/2223802897?redirect=https%3A%2F%2Ftestray.liferay.com%2Fhome%2F-%2Ftestray%2Fcase_results%2Findex%3FtestrayBuildId%3D2223801156%26testrayRunId%3D2223801860%26cur%3D1%26delta%3D200%26orderByCol%3Dstatus_sortable%26orderByType%3Dasc%26resetCur%3Dfalse)

**Comment by andreluis2022** [comment]
> Oi@andreluis2022sua solução corrige o [LocalFile.ClientExtension#ErrorAppearsWhenHTMLBlank](https://testray.liferay.com/home/-/testray/case_results/2223803799?redirect=https%3A%2F%2Ftestray.liferay.com%2Fhome%2F-%2Ftestray%2Fcase_results%2Findex%3FtestrayBuildId%3D2223801156%26testrayRunId%3D2223801860%26cur%3D2%26delta%3D200%26orderByCol%3Dstatus_sortable%26orderByType%3Dasc%26resetCur%3Dfalse) , mas quebrou outros, você poderia dar uma olhada? sorrindo_cara_com_lágrima
> 
> [LocalFile.ClientExtension#HTMLElementConatinsInvlaidCharacters ](https://testray.liferay.com/home/-/testray/case_results/2223803808?redirect=https%3A%2F%2Ftestray.liferay.com%2Fhome%2F-%2Ftestray%2Fcase_results%2Findex%3FtestrayBuildId%3D2223801156%26testrayRunId%3D2223801860%26cur%3D1%26delta%3D200%26orderByCol%3Dstatus_sortable%26orderByType%3Dasc%26resetCur%3Dfalse) [LocalFile.ClientExtension#HTMLElementMustHaveOneHyphen ](https://testray.liferay.com/home/-/testray/case_results/2223802766?redirect=https%3A%2F%2Ftestray.liferay.com%2Fhome%2F-%2Ftestray%2Fcase_results%2Findex%3FtestrayBuildId%3D2223801156%26testrayRunId%3D2223801860%26cur%3D1%26delta%3D200%26orderByCol%3Dstatus_sortable%26orderByType%3Dasc%26resetCur%3Dfalse) [.ClientExtension#HTMLElementMustStartWithLowercase](https://testray.liferay.com/home/-/testray/case_results/2223802897?redirect=https%3A%2F%2Ftestray.liferay.com%2Fhome%2F-%2Ftestray%2Fcase_results%2Findex%3FtestrayBuildId%3D2223801156%26testrayRunId%3D2223801860%26cur%3D1%26delta%3D200%26orderByCol%3Dstatus_sortable%26orderByType%3Dasc%26resetCur%3Dfalse)

ok vou revisar e alterar!!

**Comment by mateusralv** [comment]
ci:test:frontend

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by mateusralv** [comment]
ci:test:frontend

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/remoteapps/ClientExtensionEntry.path [hunk 1]
```diff
@@ -75,6 +75,11 @@
 	<td>//div[contains(@class, 'alert alert-dismissible alert-danger') and contains(.,'${text}')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>REMOTE_APPS_ERROR_WITH_MESSAGE_FAIL</td>
+	<td>//div[contains(@class, 'form-feedback-item form-validator-stack help-block') and contains(.,'${text}')]</td>
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/remoteapps/ClientExtensionEntry.path:80]:
```suggestion
	<td>WARNING_SPECIFIC</td>
	<td>//div[contains(@class, 'form-feedback-item') and contains(.,'${key_text}')]</td>
```

**Review by mateusralv — APPROVED** [review]

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
@mateusralv

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
@mateusralv

**Comment by mateusralv** [comment]
ci:test:frontend

**Comment by mateusralv** [comment]
ci:forward

**Pull request closed without merging**